code/modules/power/supermatter/supermatter.dm

```diff
@@ -141,10 +144,14 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/tritiumcomp = 0
 	var/bzcomp = 0
 	var/pluoxiumcomp = 0
+	var/h2ocomp = 0
```

Comment:
code doc these

Reply:
there is the code doc on the n2comp `// raw composition of each gas in the chamber, ranges from 0 to 1`

Reply:
fineeeeeeeeeeeeeeeeeeeeeeee

Reply:
I've done the value you requested and the ones directly related to it. I will cover the rest in a separate pr.

Reply:
it's non negotiable to document vars you are adding, it must be done.

Reply:
Understood, I was refering to the sm as a whole. I will doc the other vars that were added here.